tests/Unit/Syntax/QueryStringTest.php

```diff
@@ -19,6 +19,7 @@ public function test_it_builds_the_right_query(): void
                 'query' => 'test',
                 'default_operator' => 'OR',
                 'boost' => 1.0,
+                'fields' => [],
```

Comment:
Isn't it nicer (and maybe backwards compatible for other users) if the entire fields parameter is omitted if empty?